paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletScreen.kt

```diff
@@ -732,6 +779,28 @@ private fun AlertMessage(
     )
 }
 
+// We can't use FC's `rememberFinancialConnectionsSheet` because it doesn't support FC Lite.
+@Composable
+private fun rememberFinancialConnectionsSheetInternal(
+    financialConnectionsAvailability: FinancialConnectionsAvailability?,
+    callback: (FinancialConnectionsSheetResult) -> Unit
+): FinancialConnectionsSheetForDataLauncher? {
+    financialConnectionsAvailability ?: return null
+
+    val context = LocalContext.current
+    val contract = remember(context, financialConnectionsAvailability) {
+        FinancialConnectionsSheetForDataContract(
+            intentBuilder = financialConnectionsAvailability.getIntentBuilder(context)
+        )
+    }
+    val activityResultLauncher = rememberLauncherForActivityResult(contract, callback)
+    return remember(activityResultLauncher) {
+        FinancialConnectionsSheetForDataLauncher(
+            activityResultLauncher
+        )
+    }
+}
```

Comment:
good catch - when we make link standalone public we'll likely need to publish this